src/test/scala/uk/gov/hmrc/test/ui/pages/CommonPage.scala

```diff
@@ -40,6 +40,11 @@ object CommonPage extends BrowserDriver with Matchers {
       .navigate()
       .to(s"$host/your-account")
 
+  def navigateToStartYourReturnPage(): Unit =
+    driver
+      .navigate()
+      .to(s"$host/2022-Q2/start")
```

Comment:
I think they should be allowed to start this return and not be able to submit 2022-Q3,so seems to be a bug if you can't submit this

Reply:
This is fixed now